src/page-modules/assistant/server/journey-planner/index.ts

```diff
@@ -97,21 +97,23 @@ export function createJourneyApi(
         ? new Date(input.departureDate)
         : new Date();
 
+      const queryVariables = {
+        from,
+        to,
+        arriveBy: input.departureMode === 'arriveBy',
+        when,
+        modes: journeyModes,
+        cursor: input.cursor,
+        numTripPatterns: 10,
+        waitReluctance: 1.5,
+        walkReluctance: 1.5,
+        walkSpeed: 1.3,
+        transferPenalty: 10,
```

Comment:
This isn't a problem with this PR it self but something from earlier. But these might be considered magic numbers and not easy to understand that these are default settings. 

This could be extracted as own constant in the top like:

```
const DEFAULT_JOURNEY_CONFIG = {
    numTripPatterns: 10,
    waitReluctance: 1.5,
    walkReluctance: 1.5,
    walkSpeed: 1.3,
    transferPenalty: 10
}
```

and possibly comments on some of them if needed

Reply:
Someone (you) once told me
>  Leave it better than you found it

Words to live by 🤝 Thanks for the reminder